src/common/crypto_util.c

```diff
@@ -103,5 +106,22 @@ memwipe(void *mem, uint8_t byte, size_t sz)
   memset(mem, byte, sz);
 }
 
+void
```

Comment:
We should be sure to move the corresponding documentation this function had as well.